pkg/kubelet/kuberuntime/kuberuntime_container_linux_test.go

```diff
@@ -738,7 +739,11 @@ func TestGenerateLinuxContainerConfigNamespaces(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			got, err := m.generateLinuxContainerConfig(&tc.pod.Spec.Containers[0], tc.pod, nil, "", tc.target, false)
 			assert.NoError(t, err)
-			if diff := cmp.Diff(tc.want, got.SecurityContext.NamespaceOptions); diff != "" {
+			if diff := cmp.Diff(
+				tc.want,
+				got.SecurityContext.NamespaceOptions,
```

Comment:
is the need for this change a sign that the generated types are changing in ways that will break consumers?